### DISCUSSION TITLE
Why is Zellij saving commands across sessions? Any way to configure this?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [zellij-org/zellij#3490] [Q&A]
So I noticed that `zellij` has started remembering the programs ran in my session but the problem is, I would like to be able to close them or just make it forget since sometimes it can get annoying, for example

Here it remember some `/tmp` invocation of neovim, and its now failing to launch it every time
![screenshot of zellij](https://github.com/user-attachments/assets/e11d9894-f8a5-4989-8b74-ae1efa5863f6)

Also it saves some random programs I ran once. 

Any way to configure/ disable this behaviour? I rarely want to rerun the commands like this

**Comment by merikan** [comment]
Have you tried disabling viewport? https://zellij.dev/documentation/session-resurrection#pane_viewport_serialization
